harness/libraries/apptest.py

```diff
@@ -547,14 +546,214 @@ def _start_test(self,
             self.doInfoLogging(message)
 
     def _stop_test(self):
-
         pathtokillfile = self.get_path_to_kill_file()
         with open(pathtokillfile,"w") as kill_file:
             kill_file.write("")
 
         message =  "In function {function_name}, The kill file '{filename}' has been created.\n".format(function_name=self.__name_of_current_function(),filename=pathtokillfile)
         self.doInfoLogging(message)
 
+    # Used when --mode influx_log is run after a harness run
+    def _influx_log_mode(self):
+        """ Logs available tests to InfluxDB, via --mode influx_log """
+        currentdir = os.getcwd()
+        print(f"In {self.__name_of_current_function()}, cwd: {currentdir}")
+        # Run_Archive adds an odd ID on to the end, so this is a portable solution
+        testdir = self.get_path_to_test()
+        os.chdir(testdir)
+        # If Run_Archive exists, continue, else terminate because no tests have been run
+        if not os.path.exists('./Run_Archive'):
```

Comment:
@MichaelBrim  the `self.get_path_to_test()` should be the git repo test directory, where we would have `Run_Archive`, `Scripts`, and `Status` directories, if tests had run, or just `Scripts` if no tests had been run. We don't know beforehand if any tests have been run or not, so I'm not sure if we're able to use a templated path from layout_of_apps_directory, because that requires a specific JobID. This section of code is only discovering existing tests, it's not creating any new ones, and doesn't have the jobIDs of the tests yet.